Boa.Constrictor.Example/Tests/ScreenplayWebUiTest.cs

```diff
@@ -1,36 +1,37 @@
-﻿using Boa.Constrictor.Logging;
```

Comment:
The code in `Boa.Constrictor.Example` should not be modified. It carefully follows the tutorial.

I know you want to show the new Conditions, and I hate to say it, but this is not the place.

I'd like to eventually add a doc section to explain assorted features of Boa Constrictor. I think an article on out-of-the-box conditions would be awesome. That would be the best place to show examples.

Reply:
No problem at all, I'm agreed and I'll change it back. I believe it would be nice to have a doc section about all existing conditions too, great idea, if I may help with it I'll be very pleased to.